tests/jni_api.rs

```diff
@@ -13,6 +15,7 @@ use jni::{
 };
 
 mod util;
+use jni::objects::ReleaseMode::Copy;
```

Comment:
A nitpick: I think that it is usually better to import an enum instead of its members, especially for such commonly used things as "copy".

Reply:
Good point.

Another issue I have with this enum, is that I'm not sure if it's better to rename these members adding the `Back` suffix, i.e. `CopyBack` and `NoCopyBack`. That's to emphasize these modes apply at object's release, not creation.